rfcs/packaging/communicating-convergence-status.md

```diff
@@ -0,0 +1,141 @@
+  - Exception: `react-date-time` (not at all converged) is exported by the suite as of version 8, creating a cycle, but we can break this cycle as described above
+- Shift in focus to aggressively converging components, with a hard policy that they _must not_ depend on old components from either v8 or v0 (eliminating most risk of future cycles)
+
+While getting rid of `react-internal` is extra work, it seems worthwhile to get rid of the technical debt and improve the dev experience.
+
+### Open question: should `@fluentui/react-*` mean converged? (remove `react-` prefix from old packages?)
+
+See discussion later.
+
+### Add status badges to all component package readmes
+
+The badges below are generated using https://shields.io/ and link to a placeholder wiki page which will later contain explanations of each status.
+
+We definitely want badges for stability, possibly with sub-badges for convergence status:
+
+- [![in development](https://img.shields.io/badge/stability-in%20development-orange)](https://github.com/microsoft/fluentui/wiki/Package-status)
+- [![preview](https://img.shields.io/badge/stability-preview-yellow)](https://github.com/microsoft/fluentui/wiki/Package-status)
+  - Possible sub-badge: [![preview (legacy)](<https://img.shields.io/badge/stability-preview%20(legacy)-yellow>)](https://github.com/microsoft/fluentui/wiki/Package-status)
+- [![stable](https://img.shields.io/badge/stability-stable-green)](https://github.com/microsoft/fluentui/wiki/Package-status)
+  - Possible sub-badge: [![stable (legacy)](<https://img.shields.io/badge/stability-stable%20(legacy)-green>)](https://github.com/microsoft/fluentui/wiki/Package-status)
+- [![deprecated](https://img.shields.io/badge/stability-deprecated-red)](https://github.com/microsoft/fluentui/wiki/Package-status)
+
+We might also want separate badges for category/convergence/??? (naming suggestions welcome)
+
+- [![legacy](https://img.shields.io/badge/convergence-legacy-lightgrey)](https://github.com/microsoft/fluentui/wiki/Package-status)
+- [![mixed](https://img.shields.io/badge/convergence-mixed-yellow)](https://github.com/microsoft/fluentui/wiki/Package-status)
+- [![done](https://img.shields.io/badge/convergence-done-green)](https://github.com/microsoft/fluentui/wiki/Package-status)
+
+Notes:
+
+- Charting and maybe experiments would probably fall into the "preview (legacy)" category
+- Suite packages should list which components are converged or not
+
+## Discarded Solutions
+
+<!-- As you enumerate possible solutions, try to keep track of the discarded ones. This should include why we discarded the solution. -->
+
+## Open Issues
+
+<!-- Optional section, but useful for first drafts. Use this section to track open issues on unanswered questions regarding the design or proposal.  -->
+
+### Should we change anything about `react-cards`?
```

Comment:
@khmakoto I'd appreciate your input on this section in particular

Reply:
I think we should delete it from version 8. To my understanding there was one partner using it and they knew it wasn't officially released. We could still provide support in the 7.0 branch and, honestly, there's not much in terms of influx of `Card` related issues.